src/iiif/image_handling.py

```diff
@@ -10,6 +12,45 @@
 
 MALFORMED_SCALING_PARAMETER = "The scaling parameter is malformed. It should either be 'full' or in the form of '100,50'."
 
+BASE_INFO_JSON = {
+    "@context": "http://iiif.io/api/image/2/context.json",
+    "@id": "https://images.data.amsterdam.nl/iiif/2/edepot:SJ-00093-SJ10000463_00001.jpg",
+    "protocol": "http://iiif.io/api/image",
+    "width": None,
+    "height": None,
+    "sizes": [{"width": None, "height": None}],
+    "profile": [
+        "http://iiif.io/api/image/2/level2.json",
+        {
+            "formats": [],
+            "qualities": ["default"],
+            "supports": ["sizeByW", "sizeByH", "sizeByWh"]
+        }
+    ]
+}
+
+# TODO: To fill in
+# width
+# height
+# sizes
+# profile["formats"]  # contained only "jpg" in the original
+
+def content_type_to_format(content_type):
```

Comment:
deze functie onder 'parse_scaling_string" zetten zodat men van boven naar onder door kan lezen? Vooral met dit soort mini functies is dat fijn